### PR TITLE
[Accessibility] interactive window titles are now unique

### PR DESCRIPTION
**Pull request by bschnurr**
New interactive window titles are now unique
image

### CODE REVIEW

File: Python/Tests/Utilities.Python/PythonVisualStudioApp.cs [hunk 1]
```diff
@@ -317,7 +317,7 @@ public PythonVisualStudioApp(IServiceProvider site)
 
         public ReplWindowProxy GetInteractiveWindow(string title, ReplWindowProxySettings settings = null) {
             var iwp = GetService<IComponentModel>(typeof(SComponentModel))?.GetService<InteractiveWindowProvider>();
-            var window = iwp?.AllOpenWindows.FirstOrDefault(w => ((ToolWindowPane)w).Caption == title);
```
Comment (huguesv) [inline Python/Tests/Utilities.Python/PythonVisualStudioApp.cs]: there's another one of these that you missed, in the same file

File: Python/Product/Common/Strings.resx [hunk 2]
```diff
@@ -1035,8 +1035,8 @@ Your web browser will not be opened.</value>
     <value>Unknown command '{0}', use '$help' for a list of commands.</value>
   </data>
   <data name="ReplCaption" xml:space="preserve">
-    <value>{0} Interactive</value>
-    <comment>{0} is the name of the Python environment</comment>
```
Comment (huguesv) [inline Python/Product/Common/Strings.resx]: you said you like space too, should we change it?

Comment (bschnurr) [reply Python/Product/Common/Strings.resx]: yes. sorry missed that

File: Python/Tests/Utilities.Python/PythonVisualStudioApp.cs (hunk 1 above)
Comment (bschnurr) [reply Python/Tests/Utilities.Python/PythonVisualStudioApp.cs]: thanks

**Pull request merged**